lib/KeyboardAwareHOC.js

```diff
@@ -180,6 +207,48 @@ function listenToKeyboardEvents(ScrollableComponent: React$Component) {
       }, keyboardOpeningTime)
     }
 
+    scrollIntoView = async (
+      element: React.Element<*>,
+      options: ScrollIntoViewOptions = {}
+    ) => {
+      if (!this._rnkasv_keyboardView || !view) {
```

Comment:
As CI pointed out, view is not defined. There are also other a few more linting issues.